### ISSUE TITLE
Downstream artifact resolution isn't working on 2.13.6

### ISSUE THREAD

**Issue by djspiewak** [scala-js/scala-js#4504]
```
[error] (coreJS / update) sbt.librarymanagement.ResolveException: Error downloading org.scala-js:scalajs-compiler_2.13.6:1.6.0
[error]   Not found
[error]   Not found
[error]   not found: /home/runner/.ivy2/localorg.scala-js/scalajs-compiler_2.13.6/1.6.0/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scala-js/scalajs-compiler_2.13.6/1.6.0/scalajs-compiler_2.13.6-1.6.0.pom
[error] Total time: 23 s, completed Jun 8, 2021 5:35:35 PM
```

Getting things like this on several project builds. A quick sampling of PRs:

- https://github.com/typelevel/cats-effect/pull/2053
- https://github.com/typelevel/cats-effect-testing/pull/165
- https://github.com/typelevel/cats-mtl/pull/343
- https://github.com/typelevel/coop/pull/73

In all cases, the only change was upgrading sbt-scalajs from 1.5.1 to 1.6.0

**Comment by gzm0** [comment]
search.maven.ch confirms this. We seem to not have 2.13.6 in our build:

https://github.com/scala-js/scala-js/blob/99e55366170449103f9219b29e26b51c06b39e12/project/MultiScalaProject.scala#L83

**Comment by sjrd** [comment]
Gosh, I knew we were one release behind in 2.12, so I manually published for 2.12.14, but I forgot we were also one release behind in 2.13. I am publishing the compiler plugins for 2.13.16 now. They should be in Maven Central in a bit.
